README.md

```diff
@@ -23,6 +23,20 @@ Assuming that it is installed in e.g. `~/code/geant4/geant4.10.04.p02-install`,
 If you intend to use MiniScatter with Jupyter [4], do this in the shell you plan to start Jupyter before launching `jupyter-notebook`.
 You must also remember to load Geant4 before compiling and running MiniScatter.
 
+In order to use the graphical interface you need set the following Cmake upotns when installing GEANT4:
+-DGEANT4_USE_OPENGL_X11=ON 
```

Comment:
Use
```
bla bla
bla bla
```
for multiline code examples in GitHub MarkDown. Alternatively, `code` gives you an embedded code example.